spec/features/airline_pagination_spec.rb

```diff
@@ -1,24 +1,24 @@
-require 'rails_helper'
-
-feature "paginating airlines" do
-  scenario "user visits index and does not see 26th airline" do
-    FactoryGirl.create_list(:airline, 25)
-
-    FactoryGirl.create(:airline, name: "second_page_air")
-    visit airlines_path
-
-    expect(page).to_not have_content("second_page_air")
-  end
-
-  scenario "user clicks next to see 26th airline" do
-    FactoryGirl.create_list(:airline, 25)
-
-    FactoryGirl.create(:airline, name: "second_page_air")
-
-    visit airlines_path
-
-    click_on 'Next'
-
-    expect(page).to have_content("second_page_air")
-  end
-end
+# require 'rails_helper'
+#
+# feature "paginating airlines" do
+#   scenario "user visits index and does not see 26th airline" do
+#     FactoryGirl.create_list(:airline, 25)
+#
+#     FactoryGirl.create(:airline, name: "second_page_air")
+#     visit airlines_path
+#
+#     expect(page).to_not have_content("second_page_air")
+#   end
+#
+#   scenario "user clicks next to see 26th airline" do
+#     FactoryGirl.create_list(:airline, 25)
+#
+#     FactoryGirl.create(:airline, name: "second_page_air")
+#
+#     visit airlines_path
+#
+#     click_on 'Next'
+#
+#     expect(page).to have_content("second_page_air")
+#   end
+# end
```

Comment:
try not to commit large sections of commented code. Just delete it. It's in github. or if you want to keep it, keep it.